test/accountlinking/helperFunctions.test.js

```diff
@@ -3019,7 +3027,7 @@ describe(`accountlinkingTests: ${printPath("[test/accountlinking/helperFunctions
             await AccountLinking.linkAccounts(tpUser.user.loginMethods[0].recipeUserId, user.user.id);
 
             await AccountLinkingRecipe.getInstance().verifyEmailForRecipeUserIfLinkedAccountsAreVerified({
-                user,
+                user: user.user,
```

Comment:
btw this change I believe it was the only change in tests I made

Reply:
@porcellus

Reply:
I believe it was a bug in the test. We were sending the whole response from EmailPassword.signUp that contains { status, user }. 

On the function Accountlinking.verifyEmailForRecipeUserIfLinkedAccountsAreVerified there is a if input.user.isPrimaryUser that was never met since input.user was { status, user }

Reply:
maybe the verifyEmailForRecipeUserIfLinkedAccountsAreVerified needs a fix to check that user object is a proper user object

Reply:
It's fine: that should be prevented by type checking in normal use, we do not do that kind of checks in most other places. Also this is an internal util.